### PR TITLE
GET/DELETE /providers/:id

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

File: blueprint.md [hunk 1]
```diff
@@ -983,3 +983,30 @@ The key will be reused on the `token` property for every `/documents/` endpoint.
                     "document_count": 26
                 }
             }
+
+## Provider [/provders/{id}]
```
Comment (Neamar) [inline blueprint.md]: /providers

File: blueprint.md [hunk 2]
```diff
@@ -983,3 +983,30 @@ The key will be reused on the `token` property for every `/documents/` endpoint.
...
+### Get Provider [GET]
+Retrieve basic information about one provider.
+
+> * `404 Not Found`: `id` does not match a token
```
Comment (Neamar) [inline blueprint.md:1087]: 401 *2

File: blueprint.md [hunk 3]
```diff
@@ -983,3 +983,30 @@ The key will be reused on the `token` property for every `/documents/` endpoint.
...
+                "client": {
+                    "id": "53232d1dc8318cba94000042",
+                    "name": "Evernote"
+                }
+            }
+
+### Delete Provider [DELETE]
+Revokes a provider token and subsequently deletes linked documents.
+
+> * `404 Not Found`: `id` does not match a token
```
Comment (Neamar) [inline blueprint.md:1109]: 401 *2

**Comment by Neamar** [comment]
Missing aglio generation :(

File: blueprint.md [hunk 4]
```diff
@@ -983,3 +983,30 @@ The key will be reused on the `token` property for every `/documents/` endpoint.
                     "document_count": 26
                 }
             }
+
+## Provider [/provders/{id}]
+### Get Provider [GET]
+Retrieve basic information about one provider.
```
Comment (Neamar) [inline blueprint.md:1083]: It's only for Basic auth?

**Comment by rricard** [comment]
RE-RTM

**Pull request merged**